Makefile

```diff
@@ -37,7 +37,7 @@ build:
 
 build-linux:
 	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 \
-		go build -ldflags="-s -w" -o $(OUTPUT_DIR)/$(BINARY)-linux -tags netgo $(PKG)
+		go build -ldflags="-s -w" -o $(OUTPUT_DIR)/$(BINARY)-linux $(PKG)
```

Comment:
Hmm, at least [_some_ code](https://github.com/golang/go/blob/go1.14.3/src/net/cgo_stub.go#L5) uses `!cgo netgo`; I think we'd need a longer description for this.

Reply:
This notation means `(NOT cgo) OR netgo`.

Reply:
👍